openresty/Dockerfile-alpine

```diff
@@ -0,0 +1,254 @@
+# Current latest Openresty version is here.
+# Also specify the nginx core used. (Openresty, adds their own version numbers after the nginx one, so we can't use the same number for both Openresty and nginx)
+# Also specify the LUA version used. (Comes with the latest Openresty src bundle.)
+ARG OPENRESTY_VERSION="1.21.4.3" \
+    NGINX_VERSION="1.21.4" \
+    LUA_JIT_VERSION="2.1-20230410" \
```

Comment:
```suggestion
    LUA_JIT_VERSION="2.1-20231117" \
```

Reply:
Removed LUA_JIT_VERSION as this variable is not needed anymore